marlowe-playground-client/package.json

```diff
@@ -17,15 +17,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.10.2",
     "@popperjs/core": "^2.9.2",
-    "big-integer": "^1.6.50",
     "bignumber": "^1.1.0",
```

Comment:
we should eventually remove the bignumber library as there is a Browser default since 2020. This would require modifications in several places though.